File: scripts/r7passwd.php
#!/usr/bin/php
<?php
    if ($argc == 2 ) {
        $raw = $argv[1];
        $hashed = hash( 'sha256', $raw );
        echo $hashed, "\n";
    }
    else {
        echo "usage r7passwd passwd\n";
    }
?>

File: scripts/r7passwd.py
#!/usr/bin/python2

import sys
import hashlib

if ( len(sys.argv) == 2 ) :
    raw = sys.argv[1]
    hashed = (hashlib.sha256(raw)).hexdigest()
    print hashed
else :
    print 'usage: r7passwd.py password'
